app/models/concerns/fog_extensions/vsphere/server.rb

```diff
@@ -34,6 +34,10 @@ def scsi_controller_type
         scsi_controller.type
       end
 
+      def select_nic(fog_nics, nic)
```

Comment:
It's the same because the code *happens* to be implemented the same way as Ovirt in Fog. Libvirt isn't, and other providers aren't guranteed to be, so adding a *default* method in the superclass doesn't make sense IMO.